app/bundles/FormBundle/Form/Type/FieldType.php

```diff
@@ -301,39 +299,35 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             );
         }
 
-        if ($addInputAttributes) {
```

Comment:
I've just noticed the magic `$$f` naming above. That's why this value can be false/true, will update the PR.